dev-requirements.txt

```diff
@@ -0,0 +1,5 @@
+codecov==2.1.12
+flake8==3.9.2
+autopep8==1.5.7
```

Comment:
do we still need autopep8?

Reply:
Thanks, also realized I didn't remove it in Woodwork. Removed here and there.